### PR TITLE
[Feature][UI Next]Modify Layout in multiple languages

### LINKED ISSUE #7673: [Feature][UI-Next]Improve layout modules
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

Improve layout modules

### Use case

_No response_

### Related issues

Please refer to the main `issue`  [#7332](https://github.com/apache/dolphinscheduler/issues/7332).

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by labbomb**
## Purpose of the pull request
close #7673 

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Review by songjianet — DISMISSED** [review]
LGTM

**Comment by zhongjiajie** [comment]
@labbomb CI failed cause license failed. the following file license failed:

* dolphinscheduler-ui-next/src/layouts/content/index.tsx
* dolphinscheduler-ui-next/src/layouts/content/use-dataList.ts 

FYI, @labbomb @songjianet Next time you could click `detail` in CI result to see what had happened

image

And you would see error information
image

File: dolphinscheduler-ui-next/src/layouts/content/index.tsx [hunk 1]
```diff
@@ -3,67 +3,80 @@
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
  * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
+ * (the 'License') you may not use this file except in compliance with
```
Comment (zhongjiajie) [inline dolphinscheduler-ui-next/src/layouts/content/index.tsx]:
```suggestion
 * (the "License"); you may not use this file except in compliance with
```

File: dolphinscheduler-ui-next/src/layouts/content/use-dataList.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
  * contributor license agreements.  See the NOTICE file distributed with
  * this work for additional information regarding copyright ownership.
  * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the 'License'); you may not use this file except in compliance with
+ * (the 'License') you may not use this file except in compliance with
```
Comment (zhongjiajie) [inline dolphinscheduler-ui-next/src/layouts/content/use-dataList.ts]:
```suggestion
 * (the 'License'); you may not use this file except in compliance with
```

**Review by zhongjiajie — CHANGES_REQUESTED** [review]
BTW, should not change any content in the license header, if you not sure how to make adding to the license header

File: dolphinscheduler-ui-next/src/layouts/content/use-dataList.ts (hunk 2 above)
Comment (labbomb) [reply dolphinscheduler-ui-next/src/layouts/content/use-dataList.ts]: I've fixed it

**Review by songjianet — APPROVED** [review]

**Comment by zhongjiajie** [comment]
We have a fake test in E2E/User, but our PR has to pass E2E before merging. So I restart CI. ref #7656

**Review by zhongjiajie — APPROVED** [review]
LGTM

**Pull request merged**